server/src/main/java/io/deephaven/server/runner/Main.java

```diff
@@ -78,6 +79,9 @@ public static Configuration init(String[] args, Class<?> mainClass) throws IOExc
 
         log.info().append("Starting up ").append(mainClass.getName()).append("...").endl();
 
+        // Version properties to bootstrap before we load configuration for the first time
+        // VersionLookup.lookup();
+        // System.setProperty("deephaven.version", mainClass.getPackage().getImplementationVersion());
```

Comment:
Unused, remove?

Reply:
Yep, only left in for discussion's sake. I'll remove the commented-out implementation since we seem to be on the same page.